database/models.py

```diff
@@ -38,6 +38,8 @@ class Trial(Base):
     benchmark = Column(String, nullable=False)
     time_started = Column(DateTime(), nullable=True)
     time_ended = Column(DateTime(), nullable=True)
+    preemptible = Column(Boolean, default=False)
+    preempted = Column(Boolean, default=False)
```

Comment:
Can we avoid this platform specific thing leaking into our db model?

Reply:
An earlier implementation avoided this but it was much harder to get right and had uglier code. I think adding it to the db model makes for cleaner code.

Reply:
I could also have a foreign key to a GoogleCloudInstanceMetadata entity that can keep this info if you think it would be cleaner.

Reply:
I don't think this should make it harder to support local experiments by the way. Non-preemptible experiments should work without really using these fields.

Reply:
I wasn't worried about local experiments, but about the experiment abstraction being "leaky". It'd be ideal to keep this model about the experiment itself, not including information about how it was run on GCE. Having this metadata in a separate table would def be better, but perhaps we could store this data outside the DB too. Maybe we can have a chat about it.

Reply:
Changing to the seperate model is a bit complicated right now because there's an experiment running that uses the current (in this PR) schema and I want to run another one later today. I'll look into changing it when the experiment is done.

Reply:
We will always run preemptible experiments from now on, so does it makes sense to even have bool there for it.

Reply:
1. I'm not sure that is true, especially considering non-service users.
2. It is needed even when we use preemptibles because even then we can launch a limited number of nonpreemptibles.

Reply:
I started changing this and I'm not sure the complexity is worth it (especially since I already have some data using the current schema). Changing this seems to add an annoying layer of indirection for not much benefit.
I'm going to land as is, since this PR is sort of blocking my scalable measurer work